backend/llm.py

```
@@ -11,6 +12,7 @@
 import openai
 import anthropic
 import google.generativeai as genai
+from loguru import logger
```

Ruff

`loguru.logger` imported but unused

Remove unused import: `loguru.logger`

(F401)